### PR TITLE
Add exoplanet-core to conda-forge

### PR DESCRIPTION
**Pull request by mwcraig**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mwcraig** [comment]
@dfm -- can you please confirm that you are willing to be a maintainer on this package?

**Comment by dfm** [comment]
I am - Thanks for this!!!

**Comment by mwcraig** [comment]
One other questions: This should leave out `pymc3` and `jax` as in `celerite2`, correct?

**Comment by dfm** [comment]
Yes - that's right!

File: recipes/exoplanet-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cmake
```
Comment (dopplershift) [inline recipes/exoplanet-core/meta.yaml:22]: `cmake` should be listed as a build dep, not host. (Matters for cross-compiling.)

**Review by dopplershift — COMMENTED** [review]
Just one minor nit, otherwise this looks good to go.

File: recipes/exoplanet-core/meta.yaml (hunk 1 above)
Comment (mwcraig) [reply recipes/exoplanet-core/meta.yaml:22]: Done, thanks!

**Pull request merged**